src/Rules/SyntaxErrorInDibiPreparedStatementMethodRule.php

```diff
@@ -143,7 +144,7 @@ private function checkErrors(CallLike $callLike, Scope $scope, MethodReflection
         }
 
         $placeholders = [];
-        preg_match_all('#%(ex|in|i|and|or|s|t|d|~?like~?|n|lmt|ofs)#', $queryParameters[0], $placeholders, \PREG_SET_ORDER);
+        preg_match_all('#%(sN|bin|by|lmt|b|iN|f|dt|sql|ex|in|i|l|m|and|or|s|t|d|~?like~?|n|ofs|N)#', $queryParameters[0], $placeholders, \PREG_SET_ORDER);
```

Comment:
```suggestion
        // see https://dibiphp.com/en/documentation#toc-modifiers
        preg_match_all('#%(sN|bin|by|lmt|b|iN|f|dt|sql|ex|in|i|l|m|and|or|s|t|d|~?like~?|n|ofs|N)#', $queryParameters[0], $placeholders, \PREG_SET_ORDER);
```

Reply:
can you please explain what should be changed?

Reply:
just add the link above

Reply:
added link